pom.xml

```diff
@@ -47,6 +47,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <releaseProfiles> </releaseProfiles>
     <arguments> </arguments>
+    <argLine> </argLine>
```

Comment:
:ant: The space is not actually necessary;

```xml
<argLine/>
```

suffices, since the usage includes a space before the value. But this is fine.